string model_name
---
bool ok